rules/making-a-great-done-video/rule.md

```diff
@@ -53,12 +63,12 @@ For a great 'Done Video' here are the key things to remember:
 ![Figure: Good example – Clean Outlook UI](GoodExample-clean-UI.png)
 :::
 
```

Comment:
Hey @TobyChurches 

Is it missing a heading here?

The next list starts from 1... is it supposed to be a separate section from "presentation"?

Reply:
Hey @tiagov8 , No this was my bad (the IDE auto-corrected when I added the Clean UI dot points). I have just changed to be part of the same numbered list